code/_onclick/hud/HUD_element/_HUD_defines.dm

```diff
@@ -0,0 +1,24 @@
+//  values for setAlignment(var/horizontal, var/vertical) and alignElements(var/horizontal, var/vertical, var/list/HUD_element/targets) 
+#define HUD_NO_ALIGNMENT 0 						 //no alignment
+#define HUD_CENTER_ALIGNMENT 1					 //central alignment
+
+#define HUD_HORIZONTAL_WEST_OUTSIDE_ALIGNMENT 2	 //bordering west side of parent from outside
```

Comment:
shouldn't these be bitfields? What if you want to anchor to a corner?

Reply:
no, there are two types of alignment vertical and horizontal
proc that align uses two of them as arguments
if you want corner you align both horizontal and vertical to one side
Atleast it how it was created by captcha